src/_actions/TradeActions.js

```diff
@@ -131,8 +136,22 @@ export const updatePriceProposalSubscription = tradeID => {
             stop_profit: stopProfit,
             stop_loss: stopLoss,
         }).then(response => {
-            dispatch(updateTradeParams(tradeID, 'priceProposalID', response.proposal.id));
-            dispatch(updateProposalByID(response.proposal.id, response.proposal));
+            dispatch(updateTradeParams(tradeID, 'proposalError', undefined));
```

Comment:
We should investigate if putting too much logic in actions is an anti-pattern.
I think it is, but might be wrong.